assets/xml/objects/object_mm.xml

```diff
@@ -1,68 +1,80 @@
+        <Texture Name="gPostmanEyebrowTex" OutName="postman_eyebrow" Format="ci8" Width="16" Height="16" Offset="0x2650" />
+        <Texture Name="gPostmanMouthClosedTex" OutName="postman_mouth_closed" Format="ci8" Width="32" Height="16" Offset="0x2750" />
+        <Texture Name="gPostmanMouthOpenTex" OutName="postman_mouth_open" Format="ci8" Width="32" Height="16" Offset="0x2950" />
+        <Texture Name="gPostmanHandTex" OutName="postman_hand" Format="ci8" Width="16" Height="16" Offset="0x2B50" />
+        <Texture Name="gPostmanStrapsTex" OutName="postman_straps" Format="ci8" Width="32" Height="16" Offset="0x2C50" />
+        <Texture Name="gPostmanTorsoBunnyLogoTex" OutName="postman_torso_bunny_logo" Format="ci8" Width="16" Height="16" Offset="0x2E50" />
+
+        <!-- Postman Object Textures -->
+        <Texture Name="gPostmanClothingGradientTex" OutName="postman_clothing_gradient" Format="rgba16" Width="8" Height="8" Offset="0x8778" />
+        <Texture Name="gPostmanHatBrimTex" OutName="postman_hat_brim" Format="rgba16" Width="16" Height="16" Offset="0x87F8" />
+        <Texture Name="gPostmanHatBunnyLogoTex" OutName="postman_hat_bunny_logo" Format="rgba16" Width="32" Height="32" Offset="0x89F8" />
+        <Texture Name="gPostmanLetterBorderTex" OutName="postman_letter_border" Format="i8" Width="32" Height="32" Offset="0x91F8" />
+
+        <!-- Postman Display Lists -->
+        <DList Name="gPostmanTorsoDL" Offset="0x5B30" />
+        <DList Name="gPostmanHeadDL" Offset="0x5E70" />
+        <DList Name="gPostmanRightShoulderDL" Offset="0x6598" />
+        <DList Name="gPostmanRightArmDL" Offset="0x66D0" />
+        <DList Name="gPostmanRightHandDL" Offset="0x6808" />
+        <DList Name="gPostmanLeftShoulderDL" Offset="0x6A10" />
+        <DList Name="gPostmanLeftArmDL" Offset="0x6B48" />
+        <DList Name="gPostmanLeftHandDL" Offset="0x6C80" />
+        <DList Name="gPostmanWaistDL" Offset="0x6E88" />
+        <DList Name="gPostmanRightThighDL" Offset="0x6F20" />
+        <DList Name="gPostmanRightLegDL" Offset="0x7188" />
+        <DList Name="gPostmanRightFootDL" Offset="0x72F8" />
+        <DList Name="gPostmanLeftThighDL" Offset="0x7430" />
+        <DList Name="gPostmanLeftLegDL" Offset="0x7610" />
+        <DList Name="gPostmanLeftFootDL" Offset="0x7780" />
+        <DList Name="gPostmanLetterDL" Offset="0x8348" /> <!-- Original name is "mm_letter_model" -->
+        <DList Name="gPostmanBagDL" Offset="0x83E0" /> <!-- Original name is "mm_bag_model" -->
+        <DList Name="gPostmanHatDL" Offset="0x85C8" /> <!-- Original name is "mm_cap_model" -->
+        
+        <!-- Postman Limbs -->
+        <Skeleton Name="gPostmanSkel" Type="Flex" LimbType="Standard" LimbNone="POSTMAN_LIMB_NONE" LimbMax="POSTMAN_LIMB_MAX" EnumName="PostmanLimb" Offset="0x96E8" />
+        <Limb Name="gPostmanWaistLimb" Type="Standard" EnumName="POSTMAN_LIMB_WAIST" Offset="0x95F8" />
```

Comment:
We try to preserve `Offset` order to help when it comes to multi-version support. Could you please put `gPostmanSkel` back to the end of the file, and fix a lot of reordering done?